<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>loginWindow</class>
 <widget class="QDialog" name="loginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>621</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::ContextMenuPolicy::DefaultContextMenu</enum>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="loginPage">
    <widget class="QPushButton" name="registerButton">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>60</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Register</string>
     </property>
    </widget>
    <widget class="QLabel" name="emailLoginLabel">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>270</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Email:</string>
     </property>
    </widget>
    <widget class="QLabel" name="passwordLoginLabel">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>310</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Password:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="emailLoginInput">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>270</y>
       <width>261</width>
       <height>24</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="passwordLoginInput">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>320</y>
       <width>261</width>
       <height>24</height>
      </rect>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::EchoMode::Password</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="loginButton">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>380</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="exitButton">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>510</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="registerWidnow">
    <widget class="QPushButton" name="backToLoginPageButton">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>510</y>
       <width>131</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Back to Login Page</string>
     </property>
    </widget>
    <widget class="QLabel" name="surnameRegisterLabel">
     <property name="geometry">
      <rect>
       <x>238</x>
       <y>90</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Surname:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="emailRegisterInput">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>140</y>
       <width>201</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="studentMajorRegisterLabel">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>380</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Student Major:</string>
     </property>
    </widget>
    <widget class="QLabel" name="fullNameRegisterLabel">
     <property name="geometry">
      <rect>
       <x>238</x>
       <y>40</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Full Name:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="surnameRegisterInput">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>90</y>
       <width>201</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="emailRegisterLabel">
     <property name="geometry">
      <rect>
       <x>238</x>
       <y>140</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Email:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="passwordRegisterInput">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>180</y>
       <width>201</width>
       <height>24</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Password must contain:
- minimum 8 characters
- minimum 1 uppercase and 1 lowercase letter
- minimum 1 digit
- minimum 1 special charater (#?!@$%^&amp;*-)
</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::EchoMode::Password</enum>
     </property>
    </widget>
    <widget class="QLabel" name="passwordRegisterLabel">
     <property name="geometry">
      <rect>
       <x>238</x>
       <y>180</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Password:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="registerToDatabaseButton">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>510</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Register</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateOfBirthRegisterEdit">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>230</y>
       <width>110</width>
       <height>25</height>
      </rect>
     </property>
     <property name="displayFormat">
      <string>dd-MM-yyyy</string>
     </property>
    </widget>
    <widget class="QLabel" name="dateOfBirthRegisterLabel">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>230</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Date of Birth:</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="studentRegisterRadioButton">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>330</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Student</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="professorRegisterRadioButton">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>330</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Professor</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="nameRegisterInput">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>40</y>
       <width>201</width>
       <height>24</height>
      </rect>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
    <widget class="QComboBox" name="studentMajorRegisterCombo">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>410</y>
       <width>231</width>
       <height>24</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Aviation and Astronautics [AvAs]</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Chemical Engineering and Technology [ChET]</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Chemical Technology [ChTe]</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Civil Engineering [CiEn]</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Computer Engineering [CoEn]</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Electrical Engineering [ElEn]</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Finance and Accounting [FiAc]</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Logistics [Logi]</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Modern Management [MoMa]</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="professorTitleRegisterCombo">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>410</y>
       <width>121</width>
       <height>24</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>BSc</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Eng.</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>MSc</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PhD</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Prof.</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="professorSpecRegisterLabel">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>380</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Professor Specialization:</string>
     </property>
    </widget>
    <widget class="QLabel" name="professorTitleRegisterLabel">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>380</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Professor Title:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="professorSpecRegisterCombo">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>410</y>
       <width>231</width>
       <height>24</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Aviation and Astronautics [AvAs]</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Chemical Engineering and Technology [ChET]</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Chemical Technology [ChTe]</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Civil Engineering [CiEn]</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Computer Engineering [CoEn]</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Electrical Engineering [ElEn]</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Finance and Accounting [FiAc]</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Logistics [Logi]</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Modern Management [MoMa]</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="studiesYearRegisterLabel">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>290</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Studies Year:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="studentStudiesYearSpinBox">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>290</y>
       <width>111</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
